### DISCUSSION TITLE
qutebrowser doesn't show images for my HTML e-mail from Mutt

### DISCUSSION THREAD

**Discussion by Hund** [qutebrowser/qutebrowser#8003] [Q&A]
I use Mutt for my e-mail. I have added this to mailcap, to allow me to open HTML messages via qutebrowser:

```
text/html; qutebrowser -R --target private-window -s content.javascript.enabled false %s; needsterminal; nametemplate=%s.html
```

It works just fine, except for the fact that qutebrowser doesn't display the images. If I open the exact same file with Firefox, it works just fine with the images.

Am I doing something wrong?

**Comment by toofar** [comment]
Are there any errors in the developer tools console? What do the links in the HTML to the images look like? Is `content.local_content_can_access_file_urls` set to true?

**Comment by Hund** [comment]
Thanks! Duh.. I forgot to check the devtools.

It helped when I set `content.local_content_can_access_remote_urls `to true. :)

**Comment by The-Compiler** [comment]
btw, I'd recommend using `--temp-basedir` for this. Settings with `-s` are not sent to existing instances, and even if you happen to open a new instance with this, you likely wouldn't want to disable JS for all future qutebrowser windows.

**Reply by Hund** [reply]
> btw, I'd recommend using --temp-basedir for this.

I tried using `--temp-basedir` first, but then I don't get access to my settings.

> Settings with -s are not sent to existing instances [...]

I know. That's why I use those settings for my mailcap entry.

> [...] and even if you happen to open a new instance with this, you likely wouldn't want to disable JS for all future qutebrowser windows.

I'm confused here. Do you mean that the flag `-s <SETTING>` edits my default config? I assumed that the flag was only used for temporary overriding whatever default settings there is.

I have disabled JavaScript by default anyway. So there's not really an issue anyway, but it's good to know for any other setting that I would use with the flag `-S`. What do I use to temporary change a setting then?

**Reply by brianbuccola** [reply]
Hi, I've been wondering the exact same thing: what's the best way to open an HTML email (i) in a private window/temporary session, (ii) with temporary settings such as `content.javascript.enabled false` (that don't effect current windows), and (iii) with access to my normal settings? As @Hund, mentioned, using `--temp-basedir` means using the default settings, but as @The-Compiler mentioned, using a private window means either not being able to apply temporary settings (to existing instances), or having those settings persist for future windows.

**Reply by The-Compiler** [reply]
@brianbuccola there's a `--config-py` argument you can use with `--temp-basedir` as well.
